tests/components/simplisafe/test_config_flow.py

```diff
@@ -33,21 +35,8 @@ def mock_api():
     assert result["reason"] == "already_configured"
 
 
-async def test_get_configured_instances(hass):
-    """Test retrieving all configured instances."""
-    conf = {CONF_USERNAME: "user@email.com", CONF_PASSWORD: "password"}
-
-    MockConfigEntry(domain=DOMAIN, unique_id="user@email.com", data=conf).add_to_hass(
-        hass
-    )
-
-    assert len(config_flow.configured_instances(hass)) == 1
-
-
 async def test_invalid_credentials(hass):
     """Test that invalid credentials throws an error."""
-    from simplipy.errors import SimplipyError
-
     conf = {CONF_USERNAME: "user@email.com", CONF_PASSWORD: "password"}
 
     flow = config_flow.SimpliSafeFlowHandler()
```

Comment:
In a future PR I recommend to rewrite the tests to use `hass.config_entries.flow.async_init` to start the flow.

Reply:
You bet; will add to my todo list.